app/main.py

```diff
@@ -114,6 +115,19 @@ async def get_model_forecast_summaries(
         LOGGER.critical(exception, exc_info=True)
         raise
 
+@app.post('/noon_forecasts/', response_model=schemas.NoonForecastResponse)
+def get_noon_forecasts(request: schemas.StationCodeList, _: bool = Depends(authenticate)):
+    """ Returns noon forecasts pulled from BC FireWeather Phase 1 website for the specified
+    set of weather stations. """
+    try:
+        LOGGER.info('/noon_forecasts/')
+        start_date = datetime.datetime.now().replace(hour=0, minute=0, second=0)
```

Comment:
I think start date needs to be `datetime.datetime.now(tz=datetime.timezone.utc)` now that the graph can actually show the reference line with the current time